mParticle-Leanplum/MPKitLeanplum.m

```diff
@@ -279,7 +279,8 @@ - (MPKitExecStatus *)logLTVIncrease:(double)increaseAmount event:(MPEvent *)even
 }
 
 - (MPKitExecStatus *)receivedUserNotification:(NSDictionary *)userInfo {
-    [Leanplum handleActionWithIdentifier:nil forRemoteNotification:userInfo completionHandler:^{
+    NSString *notifIdentifier = [userInfo.allValues.firstObject isKindOfClass:[NSString class]] ? userInfo.allValues.firstObject : @"User Notification";
+    [Leanplum handleActionWithIdentifier:notifIdentifier forRemoteNotification:userInfo completionHandler:^{
```

Comment:
@samdozor Their method for receiving notifications now requires an identifier. It seemed wrong to just pass in the same string for every notification but this doesn't seem quiet right either. Any thoughts? @peterjenkins  as well